ateam_common/src/km_assignment.cpp

```diff
@@ -115,7 +133,9 @@ value.
 
 std::vector<int> max_cost_assignment(
   const Eigen::MatrixXd & cost_matrix,
-  bool max_cost
+  bool max_cost, 
+  std::vector<int> forbidden_x, 
+  std::vector<int> forbidden_y
```

Comment:
It's a little weird that the max and min versions of this function have different signatures. Did an issue come up with the enum idea we had the other night?

Reply:
Should be changed to use an enum now